### ISSUE TITLE
Stat block Mockups?

### ISSUE THREAD

**Issue by criticalfault** [TrueLunacy/HolostreetsLatex#5]
Do you happen to have a snippet for getting those sweet stat blocks as are used in 5th and 6th edition? If you do i'll write you some for 2nd and 3rd edition and push it in as a pull request.

Just let me know!

**Comment by TrueLunacy** [comment]
I don't, I'm afraid. Not even considered those yet, but they sound like a nice feature to have. A pull request would be appreciated, as always!

**Comment by criticalfault** [comment]
```latex
\definecolor{sr5orange}{RGB}{224,180,81}
\definecolor{sr5grey}{RGB}{89,89,91}
```
Getting close, but still can't figure out why the Hline's arent colored

![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/15347342/84b61743-5561-4722-baaf-c66041e67a8a)

```latex
\begin{table}
    \centering
    \begin{tabular}{ccccccccc}
        \rowcolor{black}
        \arrayrulecolor{sr5orange}\hline
        \textcolor{sr5orange}{B} & \textcolor{sr5orange}{A} & \textcolor{sr5orange}{R} & \textcolor{sr5orange}{S} & \textcolor{sr5orange}{W} & \textcolor{sr5orange}{L} & \textcolor{sr5orange}{I} & \textcolor{sr5orange}{C} & \textcolor{sr5orange}{E}SS\\ 
        \arrayrulecolor{sr5orange}\hline
        \rowcolor{sr5grey}
        4 & 4 & 4 (6) & 3 & 4 & 3 & 5 & 4 & 5.1 \\ \hline 
        \multicolumn{3}{l}{Initiative}&  \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Condition Monitor}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Limits}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Armor}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Skills}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Knowledge Skills}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Augments}& \multicolumn{6}{c}{}\\ 
        \multicolumn{3}{l}{Gear}& \multicolumn{6}{c}{}\\ 
    \end{tabular}
\end{table}
```
![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/15347342/6e521f2b-1e69-4280-85a9-3a35afd1ce9b)

**Comment by TrueLunacy** [comment]
I've not thrown it in to try yet, but if you use tabularx you can make the columns have the same width with the X specifier. Unrelated issue to what you're facing, but I use tabularx pretty heavily throughout the project.

**Comment by TrueLunacy** [comment]
Ah, I think I see your issue. The hlines are orange, they're just not thick enough to appear. You need to adjust `\arrayrulewidth` with `\setlength`.

**Comment by TrueLunacy** [comment]
I've been working on some things for SR5 and SR6-style statblocks, in the format of the document. They're not done yet - you can probably tell they don't look quite right - but here's a look.

![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/4973835/ecbcc37f-b769-4b6e-a2d9-e65cf7c980ad)

**Comment by criticalfault** [comment]
I'm getting close but definitely hitting a wall on the wrapping.
```latex
\begin{srsmalltable}
        \centering
        \begin{tabular}{ccccccccc}
            \rowcolor{black}
            \arrayrulecolor{sr5orange}\hline
            \textcolor{sr5orange}{B} & \textcolor{sr5orange}{A} & \textcolor{sr5orange}{R} & \textcolor{sr5orange}{S} & \textcolor{sr5orange}{W} & \textcolor{sr5orange}{L} & \textcolor{sr5orange}{I} & \textcolor{sr5orange}{C} & \textcolor{sr5orange}{E}SS\\ 
            \arrayrulecolor{sr5orange}\hline
            \rowcolor{sr5grey}
            \textcolor{white}{4} & \textcolor{white}{4} & \textcolor{white}{4 (6)} & \textcolor{white}{3} & \textcolor{white}{4} & \textcolor{white}{3} & \textcolor{white}{5} & \textcolor{white}{4} & \textcolor{white}{5.1} \\ 
             \arrayrulecolor{sr5orange}\hline
            \multicolumn{3}{l}{\cellcolor{black}\textcolor{sr5orange}{Initiative} } &  
            \multicolumn{6}{c}{7 + 1D6}\\ 
            \multicolumn{3}{l}{ \cellcolor{black}\textcolor{sr5orange}{Condition Monitor}}& \multicolumn{6}{c}{10}\\ 
            \multicolumn{3}{l}{\cellcolor{black}\textcolor{sr5orange}{Limits}}& \multicolumn{6}{c}{Physical 5, Mental 4, Social 5}\\ 
            \multicolumn{3}{l}{\cellcolor{black}\textcolor{sr5orange}{Armor}}& \multicolumn{6}{c}{Clubs 3, Perception 3, Pistols 4,
            Running 3, Unarmed Combat 4}\\ 
            \multicolumn{3}{l}{\cellcolor{black}\textcolor{sr5orange}{Skills}}& \multicolumn{6}{c}{}\\ 
            \multicolumn{3}{l}{\cellcolor{black}\textcolor{sr5orange}{Knowledge} Skills}& \multicolumn{6}{c}{}\\ 
            \multicolumn{3}{l}{ \cellcolor{black}\textcolor{sr5orange}{Augments}}& \multicolumn{6}{c}{}\\ 
            \multicolumn{3}{l}{ \cellcolor{black}\textcolor{sr5orange}{Gear}}& \multicolumn{6}{c}{}
        \end{tabular}
    \end{srsmalltable}
```
![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/15347342/73e28cf9-364a-4070-9d9a-2c6dd8c778a6)

**Comment by criticalfault** [comment]
Now if i could get the left side to line up, I think we'd be golden!
```latex
    \begin{srsmalltable}
        \centering
        \begin{tabular}{p{.5cm}p{.5cm}p{.5cm}p{.5cm}p{.5cm}p{.5cm}p{.5cm}p{.5cm}p{.5cm}}
            \rowcolor{black}
            \arrayrulecolor{sr5orange}\hline
            \textcolor{sr5orange}{B} & \textcolor{sr5orange}{A} & \textcolor{sr5orange}{R} & \textcolor{sr5orange}{S} & \textcolor{sr5orange}{W} & \textcolor{sr5orange}{L} & \textcolor{sr5orange}{I} & \textcolor{sr5orange}{C} & \textcolor{sr5orange}{E}SS\\ 
            \arrayrulecolor{sr5orange}\hline
            \rowcolor{sr5grey}
            \textcolor{white}{4} & \textcolor{white}{4} & \textcolor{white}{4 (6)} & \textcolor{white}{3} & \textcolor{white}{4} & \textcolor{white}{3} & \textcolor{white}{5} & \textcolor{white}{4} & \textcolor{white}{5.1} \\ 
             \arrayrulecolor{sr5orange}
             \hline
              \begin{tabular}{p{2.5cm}m{4.8cm}}
              
            \cellcolor{black}\textcolor{sr5orange}{Initiative} &  7 + 1D6\\
            \cellcolor{black}\textcolor{sr5orange}{Condition Monitor} & 10 \\
            \cellcolor{black}\textcolor{sr5orange}{Limits} & Physical 5, Mental 4, Social 5 \\
            \cellcolor{black}\textcolor{sr5orange}{Armor} & 12 \\
             \cellcolor{black}\textcolor{sr5orange}{Skills} & Clubs 3, Perception 3, Pistols 4,
             Running 3, Unarmed Combat 4 \\
             \cellcolor{black}\textcolor{sr5orange}{Knowledge Skills} & Law Enforcement 4, Local Crime 3 \\
             
             \cellcolor{black}\textcolor{sr5orange}{Augments} None & \\
             \cellcolor{black}\textcolor{sr5orange}{Gear} & Ares Predator V [Heavy Pistol, Acc 5 (7), DV
            8P, AP –1, SA, RC —, 15 (c)]
            Armor jacket[12]
            Defiance EX Shocker [Taser, Acc 4, DV
            11S(e), AP –5, SS, RC —, 4 (m)]
            Renraku Sensei commlink (Rating 3)
            Sunglasses (image link, smartlink)
            Stun baton [Club, Acc 4, Reach 1, DV 9S(e),
            AP –5, 10 charges]
            2 doses of jazz \\ 
               \end{tabular}
        \end{tabular}
    \end{srsmalltable}
```

![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/15347342/de7cac47-49de-4cea-8240-5955c3b23ff6)

**Comment by TrueLunacy** [comment]
I'd recommend using tabularx rather than just tabular, that allows you to use 'X' column specifiers and other fancy things, to fit a table into a width of your choosing, rather than having to force-set every column width.

**Comment by TrueLunacy** [comment]
Now this is more like it.

![image](https://github.com/TrueLunacy/HolostreetsLatex/assets/4973835/22bd89b9-2c51-4633-bcd6-dfdc474f5631)

**Comment by TrueLunacy** [comment]
Marking closed given no activity and the merging of #6
